lib/menu.rb

```diff
@@ -0,0 +1,32 @@
+class Menu
+
+  attr_accessor :menu
```

Comment:
Never expose internal raw data with `attr_accessor`.  You've added methods to this class to interact with the data, so it's unnecessary.